main.go

```diff
@@ -67,7 +67,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.config.Listen)
 
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
@bieli You should panic then in `main()` function in response to error returned from `start()` function. More precisely - you should use  [Fatalf()](https://golang.org/pkg/log/#Fatalf) there instead of [Printf](https://golang.org/pkg/log/#Printf).